CONTRIBUTING.md

```diff
@@ -33,6 +33,9 @@ All members of the Istio community must abide by the
 [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
 Only by respecting each other can we develop a productive, collaborative community.
 
+Please be sure to read and understand our 1 page
+[shared values](VALUES.md) summary.
```

Comment:
I'd rather if the other doc was just inline here. It's only a few paragraphs, it doesn't need to be a stand-alone thing.